project/JdkOptions.scala

```diff
@@ -26,6 +26,16 @@ object JdkOptions extends AutoPlugin {
     VersionNumber(specificationVersion).matchesSemVer(SemanticSelector(s"=1.8"))
   val isJdk11orHigher: Boolean =
     VersionNumber(specificationVersion).matchesSemVer(SemanticSelector(">=11"))
+  val isJdk17orHigher: Boolean =
+    VersionNumber(specificationVersion).matchesSemVer(SemanticSelector(">=17"))
+
+  val versionSpecificJavaOptions =
+    if (isJdk11orHigher) {
+      // for aeron
+      "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED" ::
+        // for LevelDB
+        "--add-opens=java.base/java.nio=ALL-UNNAMED" :: Nil
+    } else Nil
```

Comment:
Added here rather than CI build config to make it buildable/testable locally on JDK 17 and always passed to the multi jvm tests when on JDK 17 and later.

Reply:
The code says `isJdk11orHigher`?

Reply:
ups, should ofc be isJdk17OrHigher